src/WebJobs.Script/Utility.cs

```diff
@@ -33,6 +33,11 @@ public static string GetSubscriptionId()
             return null;
         }
 
+        public static bool IsDynamic()
+        {
+            return ScriptSettingsManager.Instance.GetSetting(EnvironmentSettingNames.AzureWebsiteSku) == ScriptConstants.DynamicSku;
```

Comment:
It uses the same approach, but with this little utility method: https://github.com/Azure/azure-webjobs-sdk-script/blob/dev/src/WebJobs.Script/Host/ScriptHost.cs#L1146-L1150